app/services/flat_service.rb

```diff
@@ -19,6 +19,12 @@ def delete_score(score_id)
       conn.delete("/v2/scores/#{score_id}")
     end
 
+    def add_collaborator(score_id, user_id)
+      conn.post("/v2/scores/#{score_id}/collaborators") do |request|
+        request.body = "{ 'user': #{user_id}, 'aclWrite': true }"
+      end
```

Comment:
Flat recognizes 3 levels of collaborators: those with ability to view/download, those with ability to write, and those with admin ability. By passing the 'aclWrite: true' param, we're giving collabs the view/download & write ability, but not full admin ability. We can adjust this as needed, for example if they need delete ability.

Reply:
I think for where were at right now this makes  lot of sense